config/locales/event_types/en.yml

```diff
@@ -3,7 +3,7 @@ en:
   event_types:
     account_created: Account created
     account_verified: Account verified
-    authenticated_at: Signed in at %{service_provider}
+    authenticated_at_html: Signed in at %{service_provider}
```

Comment:
(mentioned this in slack but leaving it here for posterity)

I think we should split out a second key for `_html` rather than rename the existing key. Let's have one key we use for HTML interpolation (generating a link via `link_to` which will escape content inside the HTML correctly) and then a second one for "plain" interpolation without the link tag. This will protect us from XSS via malformed SP names